src/weekly/WeeklyExporter.js

```diff
@@ -99,13 +99,12 @@ export class AusExporter extends WeeklyExporter {
 }
 
 export class CaExporter extends WeeklyExporter {
-  handDelivery : string
```

Comment:
I noticed a number of Canada weekly subs with the hand delivery empty so I assume that yes means hand delivery and empty/everything else means non hand delivery?